bot.rb

```diff
@@ -394,5 +402,11 @@ def collect_pr_references_from_line(line, release_name, prs_to_releases)
         end
       end
     end
+
+    def fix_checkboxes(text)
+      new_text = text.gsub(/^- \[\s*\S+\s*\]/, "- [x]")
+
+      return new_text if new_text != text else nil
```

Comment:
What the hell, that works?!

Reply:
Why not?

Reply:
It's a ternary operator [using keywords instead of operators](https://stackoverflow.com/a/4252945/865175).

Reply:
Interesting, makes sense. Although the order is off compared to the ternary operator:

```ruby
return (new_text != text ? new_text : nil) 
```

Reads kind of like a sentence this way, though. I like it.

Reply:
Also from that link I would've thought the order would need to be 

```ruby
return if new_text != text then new_text else nil
```

Reply:
Oh, right, I got it mixed up with [Python's way](https://stackoverflow.com/a/2802748/865175). I don't know how it worked though :|

Reply:
It looks like `else nil` was somehow ignored, because [this](https://stackoverflow.com/a/42308805/865175) is the correct syntax.